.github/workflows/this.release.yaml

```diff
@@ -0,0 +1,13 @@
+name: Update Major Version Tag
```

Comment:
Unless the naming scheme here is special, maybe `this-release.yaml` instead? Using periods for word separators feels a bit non-standard.